src/collections/utils.ts

```diff
@@ -75,18 +74,17 @@ function getBaseAPIPath(pathFromUrl?: string | null) {
  * @returns URL as string
  */
 export function createAstraUri (
-    databaseId: string,
-    region: string,
-    keyspace: string,
+    apiEndPoint: string,
+    keyspace?: string,
     applicationToken?: string,
```

Comment:
Rename to `apiToken`. Also, should this parameter be mandatory?

Reply:
Please disregard the renaming part, `applicationToken` is the correct name.

Reply:
sure